frontend/dockerfile/linter/ruleset.go

```diff
@@ -105,4 +105,11 @@ var (
 			return fmt.Sprintf("Multiple %s instructions should not be used in the same stage because only the last one will be used", instructionName)
 		},
 	}
+	RuleLegacyKeyValueFormat = LinterRule[func(cmdName string) string]{
```

Comment:
We should provide some hint that `key=value` should be used instead in the message. Most users will not know what is "legacy format"

Reply:
Changed this message to read differently so now it explicitly mentions the solution in the detailed description.